### PR TITLE
[16.0][FIX] sale_exception_holidays_public: tests don't pass

### PR DESCRIPTION
**Pull request by EmilioPascual**
Before, if there are less than 10 days left to the end of the year, tests don't pass

MT-4381

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
<del>This smells strongly to end of year glitch :santa: </del>

Oh, I didn't read the PR comment :sweat_smile:

File: sale_exception_holidays_public/tests/test_sale_exception.py [hunk 1]
```diff
@@ -22,7 +21,7 @@ def setUpClass(cls):
         # Create holidays
         today = fields.Date.today()
         this_year = today.year
```
Comment (chienandalu) [inline sale_exception_holidays_public/tests/test_sale_exception.py]: Maybe you just need to change this value to `holiday_date.year` and you keep the former test logic

Comment (EmilioPascual) [reply sale_exception_holidays_public/tests/test_sale_exception.py]: You're right, I wanted to make a quick fix and didn't see it 😅. Changed

File: sale_exception_holidays_public/tests/test_sale_exception.py [hunk 2]
```diff
@@ -20,11 +19,9 @@ def setUpClass(cls):
         cls.holiday_model.search([]).unlink()
 
         # Create holidays
-        today = fields.Date.today()
-        this_year = today.year
-        holiday_date = today + timedelta(days=10)
+        holiday_date = fields.Date.today()
```
Comment (chienandalu) [inline sale_exception_holidays_public/tests/test_sale_exception.py]: Well, I meant to keep this as it was as well :)
```suggestion
holiday_date = fields.Date.today() + timedelta(days=10)
```

**Review by yajo — APPROVED** [review]

**Comment by rafaelbn** [comment]
Please @leemannd @simahawk @santostelmo this test are making all PR in OCA will fails until 1st january! 😲 

Could you please review?

File: sale_exception_holidays_public/tests/test_sale_exception.py (hunk 2 above)
Comment (rafaelbn) [reply sale_exception_holidays_public/tests/test_sale_exception.py]: This suggestion @EmilioPascual looks interesting! 😄

**Comment by jbaudoux** [comment]
Add the suggested change https://github.com/OCA/sale-workflow/pull/2868#discussion_r1434953978 and then it can be merged

**Comment by ValentinVinagre** [comment]
> ~This smells strongly to end of year glitch 🎅 ~
> 
> Oh, I didn't read the PR comment 😅

we have been bad and they bring us coal 😢

**Comment by chienandalu** [comment]
> we have been bad and they bring us coal 😢

It's the Grinch of datetime issues spoiling our CIs on christmas season :grin:

**Review by chienandalu — APPROVED** [review]
Thanks :)

**Review by HaraldPanten — APPROVED** [review]

**Pull request merged**